tips/TIP-0028/event-api.yml

```diff
@@ -0,0 +1,1103 @@
+      message:
+        $ref: '#/components/messages/Block'
+  blocks/transaction/tagged-data:
+    subscribe:
+      tags:
+        - name: blocks
+        - name: transactions
+      operationId: blocksTransactionsWithTaggedData
+      description: Publishes newly received blocks containing a Tagged Data payload inside transactions.
+      message:
+        $ref: '#/components/messages/Block'
+  blocks/transaction/tagged-data/{tag}:
+    parameters:
+      tag:
+        description: Hex encoded tag of the Tagged Data Payload inside the Transaction Payload.
+        schema:
+          type: string
+          example: "0xe45"
+    subscribe:
+      tags:
+        - name: blocks
+        - name: transactions
+      operationId: blocksTransactionsWithTag
+      description: Publishes newly received blocks containing a Tagged Data payload inside transactions with a specific tag.
+      message:
+        $ref: '#/components/messages/Block'
+  blocks/tagged-data:
+    subscribe:
+      tags:
+        - name: blocks
+      operationId: blocksWithTaggedData
+      description: Publishes newly received blocks which contain tagged data payloads (encoded in hex) in their serialized binary form.
+      message:
+        $ref: '#/components/messages/Block'
+  blocks/tagged-data/{tag}:
+    parameters:
+      tag:
+        description: Hex encoded tag of the Tagged Data Payload.
+        schema:
+          type: string
+          example: "0xe45"
+    subscribe:
+      tags:
+        - name: blocks
+      operationId: blocksWithSpecificTag
+      description: Publishes newly received blocks which contain tagged data payloads with the specified tag parameter (encoded in hex) in their serialized binary form. Untagged data blocks do not get published.
+      message:
+        $ref: '#/components/messages/Block'
+  block-metadata/{blockId}:
+    parameters:
+      blockId:
+        description: Hex encoded identifier of the block.
+        schema:
+          type: string
+          example: "0xcf5f77d62285b9ed8d617729e9232ae346a328c1897f0939837198e93ec13e85"
+    subscribe:
+      tags:
+        - name: blocks
+      operationId: blockSpecificMetadata
+      description: Publishes metadata of a particular block whenever its metadata changes.
+      message:
+        $ref: '#/components/messages/BlockMetadata'
+  block-metadata/referenced:
+    subscribe:
+      tags:
+        - name: blocks
+      operationId: blockMetadataReferenced
+      description: Publishes metadata of a block whenever it gets referenced.
+      message:
+        $ref: '#/components/messages/BlockMetadata'
+  transactions/{transactionId}/included-block:
+    parameters:
+      transactionId:
+        description: Hex encoded identifier of the transaction.
+        schema:
+          type: string
+          example: "0xd026f8b1c856d4e844cc734bbe095429fb880ec4d93f3ccffe3b292a7de17be7"
+    subscribe:
+      tags:
+        - name: transactions
+      operationId: transactions

```

Comment:
Should the `operationId` be more specific, like `transactionIncludedBlock` ?

Reply:
`operationId` may only be used by tooling, so sure we can change it to something more specific, it doesn't affect client implementation as of now.

Reply:
anyway, changed in https://github.com/iotaledger/tips/pull/66/commits/796c6a7cf0f07834886d6dfcff723430ee0c685a